source/seedsource_core/django/seedsource/management/commands/calculate_zone_transfers.py

```diff
@@ -133,22 +131,34 @@ def _create_elevation_service(self, zone, band, data, nodata_value, coords):
         with Dataset(abs_path, "w", format="NETCDF4") as ds:
             coords.add_to_dataset(ds, "longitude", "latitude")
             data_var = ds.createVariable(
-                "data", data.dtype, dimensions=("latitude", "longitude"), fill_value=nodata_value,
+                "data",
+                data.dtype,
+                dimensions=("latitude", "longitude"),
+                fill_value=nodata_value,
             )
             data_var[:] = data
             set_crs(ds, "data", Proj("epsg:4326"))
 
         # extract out unmasked data
         masked_data = data[data != nodata_value]
         renderer = StretchedRenderer(
-            [(masked_data.min().item(), Color(46, 173, 60),), (masked_data.max().item(), Color(46, 173, 60),),]
+            [
+                (
+                    masked_data.min().item(),
+                    Color(46, 173, 60),
+                ),
+                (
+                    masked_data.max().item(),
+                    Color(46, 173, 60),
+                ),
+            ]
         )
 
         service = Service.objects.create(
             name=elevation_service_name,
             description="Elevation for zone {}, {} - {}".format(zone.name, low, high),
             data_path=rel_path,
-            projection="epsg:4326",
+            projection="+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs",
```

Comment:
This is the same projection, but the former style wasn't valid for the way we create the `Proj` object when rendering and so these were rendering blank for me (they used to work fine, so maybe a change in API...).